docs/site/Usage-scenarios.md

```diff
@@ -5,3 +5,13 @@ keywords: LoopBack 4.0, LoopBack-Next
 sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Usage-scenarios.html
 ---
+
+There are various usage scenario of using LoopBack, including accessing
+databases, integrating to other infrastructures and calling other services.
+
+- [create other forms of APIs](Create-other-forms-of-apis.md)
+- [access databases](Access-databases.md)
+- [call other services](Calling-other-APIs-and-Web-Services.md)
+- [integrate with infrastructures](Integrate-with-infrastructures.md)
+- [deployment](Deployment.md)
```

Comment:
I'm not sure if `deployment` belongs here. If yes, it should be the last item and the title should be more specific such as `deploy to cloud`

Reply:
+1 on `deploy to cloud`.  
`Deployment` is under `usage scenario` in the sidebar.  If it doesn't belong there, maybe we can move it somewhere.